.github/workflows/tox.yml

```diff
@@ -41,6 +41,7 @@ jobs:
           36) toxenvs="${toxenvs},coveralls,ensure_provider_tests,black,yamllint" ;;
           38) toxenvs="${toxenvs},coveralls,ansible-lint,ansible-plugin-scan,collection,ansible-test" ;;
           39) toxenvs="${toxenvs},coveralls,ansible-managed-var-comment" ;;
+          310) toxenvs="${toxenvs},coveralls,custom" ;;
```

Comment:
What is the reason that adding `custom` instead of  `ansible-managed-var-comment` here

Reply:
Other than this, lgtm.